src/main/scala/com/cognite/sdk/scala/v1/resources/threeD.scala

```diff
@@ -178,8 +177,11 @@ class ThreeDRevisions[F[_]](val requestSession: RequestSession[F], modelId: Long
       Constants.defaultBatchSize
     )
 
-  override def retrieveByIds(ids: Seq[Long]): F[Seq[ThreeDRevision]] =
-    RetrieveByIds.retrieveByIds(requestSession, baseUri, ids)
+  def retrieveById(id: Long): F[ThreeDRevision] = {
+    implicit val errorOrItemsDecoder: Decoder[Either[CdpApiError, ThreeDRevision]] =
+      EitherDecoder.eitherDecoder[CdpApiError, ThreeDRevision]
+    requestSession.get(uri"$baseUri/$id", (value: ThreeDRevision) => value)
```

Comment:
I think it should be ok to leave out the type, `value => value`

Reply:
Good point and maybe we should do this everywhere. We have been specifying the types in `[R, T]` instead, but it is longer and more verbose. For now, let's do it that way here as well, and change it everywhere in another PR.